Command/RecomputeCommand.php

```diff
@@ -125,15 +125,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         if (!$input->getOption('continue') || $input->getOption('cron')) {
-            $forceFlag = $input->getOption('force');
+            $forceFlag = (bool) $input->getOption('force');
```

Comment:
Is it not better like this?
```suggestion
            $forceFlag = $input->getOption('force') === true;
```